Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mhart/alpine-node:5.0
+
+ENV BUILD_PACKAGES bash curl-dev ruby-dev build-base libffi-dev libpq postgresql-dev git
+ENV RUBY_PACKAGES ruby ruby-io-console ruby-bundler ruby-raindrops ruby-nokogiri
```

Comment:
You probably don't need libffi, libpq, postgresql-dev and maybe not even git :)

Raindrops is for the unicorn gem, which I don't think we use in this project.